README.md

```diff
@@ -95,6 +94,7 @@ Want to work at a higher level while maximising style re-use? Check out  🍨 [S
   - [createTheme](#createtheme)
   - [createGlobalTheme](#createglobaltheme)
   - [createThemeContract](#createthemecontract)
+  - [createGlobalThemeContract](#createglobalthemecontract)
```

Comment:
Nice catch 👍

Reply:
A VSCode extension found it lol

Reply:
Neat. Which extension?

Reply:
https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

However I disabled it because it's a bit too opinionated lol